examples/flow/stokesNonNewtonian.ipynb

```diff
@@ -0,0 +1,498 @@
BFCpVIRk5+fnE1IeEhKCU6dOkR+pra0teDweiZzq6upCKpVCpVJBX1+frLpu3rwJExMTWFpawsTEhGgGAoEAPT09KCkpwbZt2+Dn50eAwdixY6FUKrU0Mblu+qtXr8LV1ZWqAJMnT6YSNZeAT5o0CSYmJnj06BGsra1RV1dHnKvKyko4OTmhq6sL/v7+WLlyJRobGyEQCGBqaopdu3bRvKKjo0OIlZOTE7Kzs0k2yMTERItrym0MOakMQI2UjkbX4JKtp+kV/rvxH5FscdwmR0dHUm0GQDcb0F4Eenp6UFxcPII86uDgQBNGQ0ODVhmAMUaeTd999x08PT1RWVmJ8ePH486dO9i4cSNsbW3x8OFDmJmZobKyEjo6OsjLy4OlpSVu3ryJoKAgGqyAGo2LjIzUmihbW1uRn5+Pzs5OTJ48GYODg7h79y7s7OwgEonA5/NhYmKC5cuXo62tDXl5eSgqKsIzzzwDhUKBM2fOUDIHqCead999F/r6+rh79y5qa2vh6+tLMg+cESmg5j2Ym5vD09MTHR0duHHjBsaMGQNbW1ukpaURNM8NQpVKhcTERMhkMpw9exbBwcFwdHTEtWvXCD5XKpXYu3cv+vv7MXPmTPB4PBw8eBDBwcEYM2YMkRpVKhUqKytx9uxZiEQieHl5wdTUlOQgAgMDce/ePUgkEnz//fcYHh4m37P+/n7ExsbiwIEDWLNmDaKjo9HR0YHIyEgcPnwY06ZNQ3d3N7Zu3YrnnnsOixYtwqFDhwhmBtQ7litXriAhIQE8Ho+Q0pdffhn5+fmoqamBsbExlSJ7e3sRGhoKiUSCzMxMhIeHY9asWVAqlfRApqSkEM/Aw8MDBgYGVNZev349AgIC4OjoiPj4eFy+fJn4C9bW1pQscuObiw8++EBLPoIb15rJFgdjr1ixQkuIl0PQNBGCx48fE7+PCxMTE9rVvv7660hKSqK/FRQUYM+ePfR7YmIiFAoFpk2bhq1bt45KsueQv9WrV8PX1xeTJ08etYTIGKPvwe28ly5d+l+aufJ4PLz//vtISUn5l7zI3nnnHVqQ/iy4hor/SWj+/2hCzID6fsyePZvUsQ8fPowPPvhgVKJ9TEwMCgsLMXbsWKhUKrzwwgt47733yMOUC6FQiK+++gqHDh0ilXJNwrrm3GdqaoqLFy/Czc2Nki2pVAobG5sRQqkBAQHYsmULgD9PtgC1cTSXEGguSJrJFuclamtriz179kAmkyErK0troevu7oa5uTn+9re/AVDPm/fu3cPw8DAuXLigdc+bmprg7OyslWxx1zEvLw+2trZkKs4JEHN8G+CfyFZLSwtJHmj6iHLt+/fu3aONj5ubG9LT0zFt2jQcPHgQ7e3tuHPnDiXlfX19tEEJDg7GxYsX4e3tTdeLU1TX0dEBY4yua0hICM6fPw+RSASJRAKpVIqqqioYGBjA399fiyoQFxeHkpISNDU1QSqVwsXFBcXFxZDJZIiMjMSdO3doLpgxYwa6urpw48YN+Pn5YcyYMSgqKkJDQwMaGhpw69YtKBQKiMVixMTEYHh4GDKZDF5eXrCxsUFBQQH4fD6mT59Oyvb6+vqIiooiNIhDC2/fvg2FQoGoqChcvXoVQ0ND0NXVhY2NDW0qXV1d0dTUhNbWVpSWliIkJIToEY6OjmCMwdbWlgCHR48ewdPTE7dv34aPjw8OHjwIxhicnJxQWVlJ4yEpKQnjx48nFwRXV1e0t7fTdbOzs4OxsTGkUilMTU3x1VdfYc+ePWhra4NcLtdC4x4/fvxUBfknpR9iYmKoKvS/Gf8RyZZEIoG+vj6RjbmLfePGDdohcjcEUC8Y1dXVIwxcHRwcaBJqaGhAU1MTQkNDiSvCuaUfPHiQBOqcnZ0JouWydz8/PxQXF9PgzszMRH9/PwYHB0kd18DAACUlJYiMjMTQ0BDc3Nygr68PIyMjyOVyrF+/Hn5+fqiurgafz4erqyvMzMygUqnQ0dEBFxcXEn57+eWXERQUBBMTEzx+/JiMSiUSCc6fPw+BQEALFqf+HR8fj5aWFsyYMQNJSUng8/n47bffwBgjsTqOm7Vw4UL09fUhLi6OFgGO38EtIH//+9/B4/GwZMkSqFQqdHd3w9nZmbrtrl27hunTp0OpVGLPnj1gjBHCxiW1crkcSqUS3333HeLi4iAWi3H06FH09vbik08+ofvo7OyMffv2wcjICKWlpdi8eTMWLlyI69evY8GCBVi3bh0AtUbY48eP8de//hU+Pj6kG7Rw4UJ0dXWhurqaJsR33nkHkydPRk9PDyIjIwkdjYuLg42NDX755RcaB2VlZXjppZcgFAoRGRlJCJKZmRni4+Opo9Hc3Bzr1q3Db7/9Bj6fj6CgIBQWFsLAwADr1q1DUlIS2trasGrVKuqo4YQd3d3daaEbHh7G3LlzAQAXL16EQCBASkoKFAoFhEKhlp0F8M8Fo7i4WMsWhZvENZMtrpRw6dIlEvUE1N25Hh4eI8pr+fn5yMjIoHNjjGHs2LF48803sWrVKq0JamhoCDt37iROoZmZGfLz83Hq1CmtBY6LS5cuISsri343NTWl8TpaaHYKcrYx/1/GaGW/0WJwcBArV67UEqodLYaHh/Hll18iOTkZ27ZteyrnSyAQYNq0aXj11VfR3NyM1NRU1NXVjXrs/Pnz8cknn9DfGWNITEykZ4ELGxsbXL58WWsccdzBJ5OoZcuW4dlnn9Uqf4+WbAmFQjImHy3R1ExEw8PDiW+qUqm0Giu6u7thZGREzTXc8wOokyJNweaHDx/Cz88P0dHRAP5pzQaoEeyAgAD6XH19fQgEAqoEqFQqEiS9fv069PT0EBQUBHd3d0rQXnzxRQDquXH79u0wNjZGW1sb7t+/j7i4OK3SOCewnZqaCoVCQQnsxYsXYW9vT++VlpZGFQau01AgEMDLywvnz5/HxIkTce/ePXh6eiI7OxtRUVGkk8XFhAkTkJaWBgsLCzQ3NyMwMJAsvjRFVAF1
```

Comment:
@wd15 I realize this is taken directly from [`stokesCavity.py`](https://github.com/usnistgov/fipy/blob/master/examples/flow/stokesCavity.py), but why use `_FaceGradVariable` here? Does `pressure.faceGrad` not work for some reason?